incubator-tools/synonyms_based_splitter_document_labeling/synonyms_based_splitter_document_labeling.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
<p>I'm not sure I understand what this does. I understand it's auto-labeling documents for CDS, but how does the "synonyms" list fit in? How is that used to label?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Ok, can you add information to the notebook description that explains what this tool does?

Reply:
Updated